include/vctr/Containers/VctrBase.h

```diff
@@ -600,6 +600,30 @@ class VctrBase : private Config,
         return it == rend() ? std::nullopt : std::optional<size_t> (std::distance (it, rend()) - 1);
     }
 
+    /** Returns a copy of the first element value which is greater or equal to bound or std::nullopt if no such value is found.
```

Comment:
hm, tbh it does not return a copy of the first element but an optional of the copy of the first element ;)

Reply:
Agreed this could be more precise